natural-language-processing/ner.md

```diff
@@ -1,26 +1,95 @@
+
+**4.2 Efficient Search Algorithms**  
+
+Let’s suppose you are intending to design an internal search algorithm for an online publisher that has millions of articles. If for every search query the algo ends up searching all the words in millions of articles, the process will take hell lot of time! Instead, if NER can be run once on all the articles and the relevant entities (tags) associated with each of those articles are stored separately, this could speed up the search process incredibly! With this approach, a search term will be matched with only the small list of entities discussed in each article leading to faster search execution.
+
+**4.3 Customer Support**  
+
+There are a number of ways to make the process of customer feedback procedure smooth and easy by using NER. Let’s take an example to understand the process. If you are handling the customer support department of an electronic store with multiple branches worldwide, you go through a number mentions in your customers’ feedback. 
+
+**5 Conclusion**  
+
+Named Entity Recognition has been developing continuously for over 15 years. The novel use is to extract different types of information (name, date, time, location) from the text. In addition, there are more than 20 languages and more than 200 types of entities. Most researches are interested in specific information on topic types such as news articles, web page information, etc..This provides an overview of techniques for creating a NERC system, from manual rule-based assignments to providing good and accurate results. But it comes with the time it takes to set the rules like, Supervised learning requires a large corpus that has been labeled, Semi-supervised and unsupervised learning allows for rapid recognition of entities without having to have a large labeled corpus.
 
+**References:**  
+📌 To know more about NER Deep learning approaches - Refer to https://www.depends-on-the-definition.com/introduction-named-entity-recognition-python/  
+📌 More details about LSTMs can be found [here](http://colah.github.io/posts/2015-08-Understanding-LSTMs/)
+📌 An implementation of the MUC evaluation metrics can be found:
+* https://github.com/jantrienes/nereval
+* [“Automatic Content Extraction 2008 Evaluation Plan (ACE08)”](http://www.eng.utah.edu/~cs6961/papers/ACE-2008-description.pdf)
+* [“The Automatic Content Extraction (ACE) Program Tasks, Data, and Evaluation”](https://pdfs.semanticscholar.org/0617/dd6924df7a3491c299772b70e90507b195dc.pdf) 
+* http://www.aclweb.org/anthology/M93-1007  
```

Comment:
_Suggestions:_
Can you please go thru the resources (and also presentations) on https://bit.ly/better-nlp-launch and see if there are any useful resources you could pick and add to the above list.

Also one of the presentations slides list a number of components to NER via either Spacey or NLTK - would be good to include that list above somewhere.

Thanks

Reply:
Sure! Will check and update.

Reply:
Maybe via a new PR, let's merge this for now